### ISSUE TITLE
Language tool support

### ISSUE THREAD

**Issue by lisuml** [otsaloma/gaupol#41]
It would be really nice to have it available in gaupol:

https://www.languagetool.org/

**Comment by otsaloma** [comment]
This looks moderately interesting. I don't really know where this would fit though: would it be an alternative to the current spell-check? Meaning the user could choose between the current Enchant or Language Tool? Abstracting the spell-check to support multiple "engines" sounds like a lot of work. Especially if it's not similar enough to spell-checking, e.g. having overlapping grammar and spelling errors. There's a possible migration planned to gspell (#12), which would be a right time to consider changes in abstraction.

As downsides, (1) Language Tools seems to be a Java application, (2) from Python we'd probably need to use the command line program and (3) it doesn't seem to be packaged in distros.

To summarise: worth considering, but unlikely anytime soon.

**Comment by blogodorio** [comment]
This software indicates an error that I don't know what about... It says that I must send a message about some bug, but I could not find out how to fix it...
I have installed it on PC - Windows 10 - 32bits/2gb ram

![image](https://cloud.githubusercontent.com/assets/25767509/22932631/bbe7b738-f2af-11e6-81d8-36a7df13fd72.png)

**Comment by otsaloma** [comment]
@blogodorio One issue per issue please, moved: #43

**Comment by otsaloma** [comment]
By now I'll conclude this is more trouble than it's worth.
